.gitignore

```diff
@@ -10,3 +10,4 @@ docs/_build
 .coverage
 .tox
 cover
+env
```

Comment:
This is for a virtualenv, but let me know if you have another convention you strongly prefer.